### PR TITLE
Import the jenkins parent pom

### PR DESCRIPTION
**Pull request by sladyn98**
We would like to make use of the jenkins Parent pom so that we can use spot bugs etc.

### CODE REVIEW

**Comment by kwhetstone** [comment]
Adding some notes from the channel: we might end up skipping this depending on feedback.  We might just have to add the build to Jenkins.ci in general in the tools section.

**Comment by sladyn98** [comment]
@kwhetstone Is there a way to add a manual build ?

File: pom.xml [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.3.0.RELEASE</version>
-		<relativePath/> <!-- lookup parent from repository -->
-	</parent>
-	<groupId>io.jenkins.tools.distribution-service</groupId>
+  <parent>
+    <groupId>org.jenkins-ci</groupId>
+    <artifactId>jenkins</artifactId>
+    <version>1.51</version>
+  </parent>
```
Comment (LinuxSuRen) [inline pom.xml:9]: How about formatting these code lines?

File: pom.xml [hunk 2]
```diff
@@ -42,6 +41,18 @@
...
+    <dependencies>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
+        <version>2.2.6.RELEASE</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```
Comment (LinuxSuRen) [inline pom.xml:54]: Please keeping the same indent pattern.

**Comment by LinuxSuRen** [comment]
> @kwhetstone Is there a way to add a manual build ?

@sladyn98 You can do some research work about GitHub bot.

**Comment by sladyn98** [comment]
> @sladyn98 You can do some research work about GitHub bot.

Do you mean github actions ? I was referring it to run on `ci.jenkins.io` because the build does not show up there

**Comment by LinuxSuRen** [comment]
Not GitHub actions. Something like this https://prow.k8s.io/command-help.

**Comment by sladyn98** [comment]
We do not need this PR because we already have the jenkinsfile ready.

**Pull request closed without merging**